src/PlusDataCollection/vtkPlusGenericSerialDevice.cxx

```diff
@@ -56,6 +56,8 @@ vtkPlusGenericSerialDevice::vtkPlusGenericSerialDevice()
   : Serial(new SerialLine())
   , SerialPort(1)
   , BaudRate(9600)
+  , DTR(false)

```

Comment:
Could the variable be more descriptive? Does this mean the requested value of the DTR/DTS pin? If so, something like "RequestedDTR" or "RequestedDTRPinState"